app/views/layouts/modular/application.html.erb

```diff
@@ -12,8 +12,10 @@
         <%= stylesheet_link_tag "modular/vendor.css", media: "all" %>
         <%= stylesheet_link_tag "modular/app-blue.css", media: "all" %>
         <%= stylesheet_link_tag "modular/custom.css", media: "all" %>
+        <%= yield :stylesheets %>
         <%= csrf_meta_tags %>
         <!-- Additional scripts -->
+        <link href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css" rel="stylesheet" integrity="sha384-wvfXpqpZZVQGK6TAh5PVlGOfQNHSoD2xbE+QkPxCAFlNEevoEH3Sl0sibVcOQVnN" crossorigin="anonymous">
```

Comment:
ideally in the future let's package those ourselves, there is no real benefit in using a CDN for our admin stuff